### PR TITLE
OSDOCS#6417: Adding RN for custom RHCOS image for Azure cluster

### PR DESCRIPTION
**Pull request by mjpytlak**
Version(s):
4.14

Issue:
This PR addresses the release note for [osdocs-6417](https://issues.redhat.com/browse/OSDOCS-6417).

Link to docs preview:

[Using a custom RHCOS image for a Microsoft Azure cluster](https://66006--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-azure-custom-rhcos)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by mjpytlak** [comment]
@jinyunma PTAL. Thank you.

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -141,6 +141,27 @@ Before you set a path value for the `template` parameter, ensure that the defaul
...
+* `compute.platorm.azure.osImage.sku`
+* `compute.platorm.azure.osImage.version`
+* `controlPlane.platorm.azure.osImage.publisher`
+* `controlPlane.platorm.azure.osImage.offer`
+* `controlPlane.platorm.azure.osImage.sku`
+* `controlPlane.platorm.azure.osImage.version`
+* `platform.azure.defaultMachinePlatform.osImage.publisher`
+* `platform.azure.defaultMachinePlatform.osImage.offer`
+* `platform.azure.defaultMachinePlatform.osImage.sku`
+* `platform.azure.defaultMachinePlatform.osImage.version`
```
Comment (jinyunma) [inline release_notes/ocp-4-14-release-notes.adoc:160]: missed new field introduced in 4.14 `azure.osImage.plan` under compute/controlPlane/defaultMachinePlatform.
```
$ ./openshift-install explain installconfig.platform.azure.defaultMachinePlatform.osImage.plan
KIND:     InstallConfig
VERSION:  v1

RESOURCE: <string>
  Valid Values: "WithPurchasePlan","NoPurchasePlan"
  Plan is the purchase plan of the image. If omitted, it defaults to "WithPurchasePlan".

```

Comment (mjpytlak) [reply release_notes/ocp-4-14-release-notes.adoc:160]: > missed new field introduced in 4.14 `azure.osImage.plan` under compute/controlPlane/defaultMachinePlatform.

@jinyunma When sizing this effort with @r4f4, we agreed that it was best to leave this parameter undocumented. [1]

[1] https://issues.redhat.com/browse/CORS-2503?focusedId=22420589&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-22420589

Comment (jinyunma) [reply release_notes/ocp-4-14-release-notes.adoc:160]: thanks for the information, then the update LGTM.

**Review by skrthomas — APPROVED** [review]
/lgtm

**Pull request merged**